source/api/presentation/3.0/index.md

```diff
@@ -240,6 +240,25 @@ The value _MUST_ be a JSON array, with each item in the array being a JSON objec
 {"thumbnail": [{"id": "https://example.org/img/thumb.jpg", "type": "Image"}]}
 ```
 
+##### posterCanvas
+One or more Canvases providing content associated with the object represented by this resource but not part of that object. Examples include an image to show while a duration-only Canvas is playing audio; images, text and sound standing in for video content before the user initiates playback; or a film poster to attract user attention. The content provided by `posterCanvas` differs from a thumbnail: a client might use `thumbnail` to summarise and navigate multiple resources, then show content from `posterCanvas` as part of the presentation of a single resource. A poster Canvas can have different dimensions to the Canvas(es) of this resource.
```

Comment:
> A poster Canvas can have different dimensions to the Canvas(es) of this resource.

A poster Canvas can have different dimensions to those of the Canvas(es) of this resource.